packages/devextreme/js/__internal/ui/chat/chat_avatar.ts

```diff
@@ -27,20 +30,73 @@ class Avatar extends Widget<Properties> {
 
     super._initMarkup();
 
-    this._renderInitialsElement();
-    this._updateInitials();
+    this._renderAvatarContent();
+  }
+
+  _renderAvatarContent(): void {
+    this._$content?.remove();
```

Comment:
I would prefer not to use a reference to an element here and, if possible, avoid using it to make the functions pure. You will have to expand this functionality further, and you might arrive at such a solution in future PRs.